test/check-application

```diff
@@ -1290,8 +1294,8 @@ class TestApplication(testlib.MachineCase):
         b.wait_not_present("#run-image-dialog-pull-latest-image")
 
         # Create Container, image is pulled and should end up being "running"
-        b.click('.pf-c-modal-box__footer #create-image-create-run-btn')
-        sel = "> span:not(.downloading)"
+        b.click('.pf-c-modal-box__footer button:contains("Create and run")')
```

Comment:
What makes you prefer using `:contains`?

Reply:
bad rebase? :thinking: